### PR TITLE
corrected some typo

### PR DESCRIPTION
**Pull request by yaggit**
# Description

Some spelling mistakes correction and readability improved.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,11 +84,11 @@ And much more!
 
 ## :question: How to use it
 
-If you need some help, [the best place to start is this article on dev.to](https://dev.to/johannesjo/getting-started-with-super-productivity-2791).
+If you need some help, [this aarticle on dev.to is the best place to start](https://dev.to/johannesjo/getting-started-with-super-productivity-2791).
```
Comment (savornicesei) [inline README.md:87]: Typo `aarticle`

**Review by savornicesei — APPROVED** [review]
I'm not seeing any other typos.

**Comment by johannesjo** [comment]
Thank youu!

**Comment by yaggit** [comment]
can you please add a tag of hactoberfest-accepted ?

**Comment by johannesjo** [comment]
I know see that there is a typo in  the PR. I'll revert this.

**Pull request merged**